packages/components/src/radio-control/index.native.js

```diff
@@ -0,0 +1,39 @@
+/**
+ * Internal dependencies
+ */
+import RadioCell from '../mobile/bottom-sheet/radio-cell';
+
+function RadioControl( {
+	help,
+	instanceId,
+	className,
+	onChange,
+	selected,
+	options = [],
+	...props
+} ) {
+	const id = `inspector-radio-control-${ instanceId }`;
+
+	return (
+		<>
+			{ options.map( ( option, index ) => {
+				return (
+					<RadioCell
+						label={ option.label }
+						id={ id }
+						help={ help }
+						className={ className }
+						aria-describedby={ !! help ? id + '__help' : undefined }
```

Comment:
I'm wondering if we use these props? Or are they coming from the web version of the `RadioControl`?

Reply:
Removed. Basically we don't need them, they are coming from web implementation